### PR TITLE
Add machine translation model: transformer_en-de, transformer_zh-en

### PR DESCRIPTION
**Pull request by KPatr1ck**
Add transformer_en-de module
- End to end prediction: text(en) in text(de) out
- Translation serving support

### CODE REVIEW

**Comment by KPatr1ck** [comment]
Add transformer_zh-en module

File: modules/text/machine_translation/transformer/zh-en/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+### Step1: 启动PaddleHub Serving
+
+运行启动命令：
+
+```shell
+$ hub serving start -m transformer_zh-en
+```
+
+通过以上命令可完成一个英德机器翻译API的部署，默认端口号为8866。
```
Comment (nepeplwu) [inline modules/text/machine_translation/transformer/zh-en/README.md]:
```suggestion
通过以上命令可完成一个中英机器翻译API的部署，默认端口号为8866。
```

File: modules/text/machine_translation/transformer/zh-en/README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+```shell
+$ hub install transformer_zh-en==1.0.0
+```
+
+## 概述
+
+2017 年，Google机器翻译团队在其发表的论文[Attention Is All You Need](https://arxiv.org/abs/1706.03762)中，提出了用于完成机器翻译（Machine Translation）等序列到序列（Seq2Seq）学习任务的一种全新网络结构——Transformer。Tranformer网络完全使用注意力（Attention）机制来实现序列到序列的建模，并且取得了很好的效果。
+
+transformer_zh-en包含6层的transformer结构，头数为8，隐藏层参数为512，参数量为64M。该模型在[CWMT2021的数据集](http://nlp.nju.edu.cn/cwmt-wmt)进行了预训练，加载后可直接用于预测。
```
Comment (nepeplwu) [inline modules/text/machine_translation/transformer/zh-en/README.md]:
```suggestion
transformer_zh-en包含6层的transformer结构，头数为8，隐藏层参数为512，参数量为64M。该模型在[CWMT2021的数据集](http://nlp.nju.edu.cn/cwmt-wmt)进行了预训练，加载后可直接用于预测，提供了中文翻译为英文的能力。
```

File: modules/text/machine_translation/transformer/en-de/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+```shell
+$ hub install transformer_en-de==1.0.0
+```
+
+## 概述
+
+2017 年，Google机器翻译团队在其发表的论文[Attention Is All You Need](https://arxiv.org/abs/1706.03762)中，提出了用于完成机器翻译（Machine Translation）等序列到序列（Seq2Seq）学习任务的一种全新网络结构——Transformer。Tranformer网络完全使用注意力（Attention）机制来实现序列到序列的建模，并且取得了很好的效果。
+
+transformer_en-de包含6层的transformer结构，头数为8，隐藏层参数为512，参数量为64M。该模型在[WMT'14 EN-DE数据集](http://www.statmt.org/wmt14/translation-task.html)进行了预训练，加载后可直接用于预测。
```
Comment (nepeplwu) [inline modules/text/machine_translation/transformer/en-de/README.md]:
```suggestion
transformer_en-de包含6层的transformer结构，头数为8，隐藏层参数为512，参数量为64M。该模型在[WMT'14 EN-DE数据集](http://www.statmt.org/wmt14/translation-task.html)进行了预训练，加载后可直接用于预测，提供了英文翻译为德文的能力。
```

File: modules/text/machine_translation/transformer/zh-en/README.md (hunk 1 above)
Comment (KPatr1ck) [reply modules/text/machine_translation/transformer/zh-en/README.md]: 已改正。

File: modules/text/machine_translation/transformer/zh-en/README.md (hunk 2 above)
Comment (KPatr1ck) [reply modules/text/machine_translation/transformer/zh-en/README.md]: 描述已添加。

File: modules/text/machine_translation/transformer/en-de/README.md (hunk 3 above)
Comment (KPatr1ck) [reply modules/text/machine_translation/transformer/en-de/README.md]: 描述已添加。

**Pull request merged**